_posts/2018-10-10-Laborjournal.md

```diff
@@ -1,7 +1,7 @@
 ---
 site: freiburg
-title: "New article 'Datenanalyse mit dem Galaxy Server'"
-tags: [paper]
+title: "New Article 'Datenanalyse mit dem Galaxy Server'"
+tags: [article]
```

Comment:
what is the difference between article and paper?

Reply:
Articles are the ones in German that I don't understand ;) I guess that those are more outreach articles and not scientific papers. Should we label all as papers?

Reply:
Ok, so we distinguish between scientific papers and PR articles. Makes sense.